utils/nhs_number_generator.py

```diff
@@ -73,7 +73,9 @@ def random_nhs_number_generator():
         check_digit = calculate_check_digit(candidate_number)
 
         if check_digit != 10:
-            return candidate_number + str(check_digit)
+            nhs_number = candidate_number + str(check_digit)
```

Comment:
Is this left-over from some debugging?

Reply:
This is intentional in order to print out the NHS number in the console along with the prescription ID for easy lookups elsewhere